apps/examples/tanstack-start/src/styles.css

```
@@ -154,279 +225,553 @@ pre code {
+  outline: 2px solid var(--accent);
+  outline-offset: 2px;
+}
+
 .page-wrap {
-  width: min(1080px, calc(100% - 2rem));
+  width: min(1120px, calc(100% - 2rem));
   margin-inline: auto;
 }
 
 .display-title {
   font-family: 'Fraunces', Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Fraunces" (font-family-name-quotes)

(font-family-name-quotes)